providers/aws/fargate/region.go

```diff
@@ -12,6 +12,9 @@ var (
 	// FargateRegions are AWS regions where Fargate is available.
 	FargateRegions = Regions{
 		"us-east-1",
+		"us-east-2",
+		"us-west-2",
+		"eu-west-1",
```

Comment:
Did you actually do end-to-end deployment tests to these regions using a VK? I was extending the existing deployment tests before adding the new regions.